server/app/views/components/ButtonStyles.java

```diff
@@ -97,7 +96,9 @@ public final class ButtonStyles {
   public static final String LINK_STYLE =
       StyleUtils.joinStyles(
           StyleUtils.removeStyles(ApplicantStyles.LINK, "text-sm"),
-          "bg-transparent",
           "border-none",
           "hover:bg-opacity-0");
+
+  public static final String LINK_STYLE_PUBLISH_PROGRAM =
```

Comment:
Why: Adding a new style because we have some places in the application that have issues with `bg-transparent` being there and others that have issues if it isn't there.